lib/search/presenters/result_presenter.rb

```diff
@@ -2,11 +2,12 @@ module Search
   class ResultPresenter
     attr_reader :raw_result, :registries, :schema, :search_params
 
-    def initialize(raw_result, registries, schema, search_params)
+    def initialize(raw_result, registries, schema, search_params, result_rank:)
```

Comment:
Since things will break if this isn't set (comparing `nil` to a number), I think this should be a mandatory argument

Reply:
I think `result_rank` doesn't have a default value here so it's required - it will raise `ArgumentError (missing keyword: result_rank)`